spec/policies/finance_profile_policy_spec.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe FinanceProfilePolicy, type: :policy do
+  subject { described_class.new(user, finance_profile) }
+  let(:user) { create(:user, :finance) }
+  let(:finance_profile) { create(:finance_profile) }
+
+  context "being a finance" do
+    it { is_expected.to permit_action(:index) }
+  end
+
+  context "not being a finance" do
+    it { is_expected.to forbid_new_and_create_actions }
+    it { is_expected.to forbid_edit_and_update_actions }
+    it { is_expected.to forbid_action(:destroy) }
+  end
```

Comment:
What about the new, create, etc. actions for a finance user? 

What about index action for non-finance user? Also, I reckon when you add those tests they will break. I think you want your `let(:user)` inside each context :)

Reply:
Done